news/_posts/2017-11-21-eindejaarsactie.markdown

```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title:  "Eindejaarsactie"
+date:   2017-11-21 20:00:00
+description: Eindejaarsactie 2017.
+permalink: /news/2017-11-21-eindejaarsactie/
+---
+
+Naar aanleiding van de aankomende eindejaarsfeesten organiseert Basket Lummen, in samenwerking met Prik&Tik Drinkcenter Borgers en De kaarskesprocessie Leonidas Lummen, een wijn- en pralinesverkoop ten voordele van de club. 
+
```

Comment:
Ben jij ook altijd op zoek naar een leuk cadeautje voor de grootouders of meter of peter... Zoek niet verder!